CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
-## ChangeLog
+# Changelog
+All notable changes to this project will be documented in this file.
 
-## 0.1.0
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [Unreleased]
+
+## [0.2.0] - 2020-10-28
+### Added
+- Support for metrics (#10)
+
+### Changed
+- Updated to use version 0.13.0 of the go.opentelemetry.io/otel packages. (#30)
+- Standardized CHANGELOG.md format.  When making changes to this project, add human-readable summaries of what you've done to the "Unreleased" section above. When creating a release, move that information into a new release section in this document. (#35)
+
+## [0.1.0] - 2019-12-31
 First release!
```

Comment:
```suggestion
First release!

[Unreleased]: https://github.com/newrelic/opentelemetry-exporter-go/compare/v0.2.0...HEAD
[0.2.0]: https://github.com/newrelic/opentelemetry-exporter-go/releases/tag/v0.2.0
[0.1.0]: https://github.com/newrelic/opentelemetry-exporter-go/releases/tag/v0.1.0
```

Reply:
Thanks for catching this! Do you think we should change the 0.2.0 entry to a "compare/v0.1.0...v0.2.0" just like the example at https://keepachangelog.com/en/1.0.0/ ?

Reply:
Yeah, that sounds good as well. :+1: